protostar/starknet/cheatable_cached_state.py

```diff
@@ -11,6 +11,8 @@
 from protostar.starknet.types import AddressType, ClassHashType, SelectorType
 
 
+CallData = list[int]
```

Comment:
[Functionality] This cheatcode should support data transformer. Data transformer accepts `CairoOrPythonData`.

Reply:
[add function name as argument](https://github.com/software-mansion/protostar/pull/1060/commits/03936e2d2755036d14f9f9de05babd3eb807faf8)